src/commands/dsyms/README.md

```diff
@@ -0,0 +1,75 @@
+# dSYMs command
+
+Upload dSYM files to Datadog to symbolicate your crash reports.
+
+## Usage
+
+### Setup
+
+You need to have `DATADOG_API_KEY` in your environment.
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+It is possible to configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable to `datadoghq.eu`. By defaut the requests are sent to Datadog US.
+
+It is also possible to override the full URL for the intake endpoint by defining the `DATADOG_SOURCEMAP_INTAKE_URL` environment variable.
+
+
+
+### Commands
+
+#### `upload`
+
+This command will upload all dSYM files in the given directory.
+If your app has `BITCODE` enabled, more info in **Bitcode** section.
+
+To upload the dSYM files in your Derived Path, this command should be run:
+
+```bash
+datadog-ci dsyms upload ~/Library/Developer/Xcode/DerivedData/
+```
+
+In addition, some optional parameters are available:
+
+* `--concurrency` (default: `20`): number of concurrent upload to the API.
+* `--dry-run` (default: `false`): it will run the command without the final step of upload. All other checks are performed.
+
+#### Bitcode
+
+With bitcode enabled, you should download your app's dSYM files from [App Store Connect](https://appstoreconnect.apple.com/).
+They come in the form of a zip file, named `appDsyms.zip`. In that case, you can run `datadog-ci` by pointing to the zip file.
+
+```bash
+// if appDsyms.zip path: ~/Downloads/appDsyms.zip
+datadog-ci dsyms upload ~/Downloads/
```

Comment:
NIT: why not target the file directly?

Reply:
my first implementation supported targeting both folder and file path, but after some internal discussion it was decided to be confusing for the users. 
that's why i ended up with targeting folder only

Reply:
IMO it is more convenient and cleaner if we expect a list of paths (e.g. a single `.zip` file or a list of `.dsym` files) instead of a folder path. 

Given the example used above:
```bash
datadog-ci dsyms upload ~/Downloads/
```
will it unzip and inspect all `*.zip` archives in the `~/Downloads/` folder? If so, this might be needless if the user has other random archives in there.

Reply:
i agree. we can re-open this discussion

Reply:
ℹ️ we decided to support both `~/Downloads/appDsyms.zip` (file) and `~/Library/Xcode/Developer/DerivedData/` (folder) as inputs